CHANGES.md

```diff
@@ -3,15 +3,21 @@ Change Log
 
 ### 1.63 - 2019-11-01
 
-We’ve consolidated all of our website content from cesiumjs.org and cesium.com into one home on cesium.com.  Here’s where you can now find:
+##### Major Announcements :loudspeaker:
+* Cesium has migrated to ES6 modules. This may or may not be a breaking change for your application depending on how you use Cesium.  See our [blog post](https://cesium.com/blog/2019/10/31/cesiumjs-es6/) for the full details.
+
```

Comment:
Is this whitespace intentional?

Reply:
I'm just used to breaking up bullets this way (but it looks like Cesium crams them all together although CHANGES isn't fully consistent here either).  I don't think it matters one way or the other.